src/components/Menu.js

```diff
@@ -35,15 +37,13 @@ class Menu extends React.Component {
         const { menu } = this.state
 
         return (
-            <div className="menu">
-                <StyleMenu className="menu__list">
-                    {menu.map((item, index) => (
-                        <li key={index} className="menu__item">
-                            <a className="menu__link" href="#">{item.value}</a>
-                        </li>
-                    ))}
-                </StyleMenu>
-            </div>
+            <StyleMenu>
+                {menu.map((item, index) => (
+                    <li key={index} className="menu__item">
+                        <a className="menu__link" href="#">{item.value}</a>
```

Comment:
I recommend to put a menu item's URL into the data structure:

```jsx
const menu = [
  { value: 'foo', url: '#' },
  ...
]

...

<a className="menu__link" href={item.url}>{item.value}</a>
```

This way _all_ the data about your menu is in the JSON (array).